### PR TITLE
WebBrowser: console.log in scripts now also logs to inviwo logggers

### PR DESCRIPTION
**Pull request by r-englund**
Also removed some warnings.

### CODE REVIEW

File: modules/webbrowser/src/webbrowserclient.cpp [hunk 1]
```diff
@@ -227,4 +226,41 @@ void WebBrowserClient::OnLoadError(CefRefPtr<CefBrowser> browser, CefRefPtr<CefF
...
+            replaceInString(src, "\\", "/");
+            file = splitString(src, '/').back();
+        }
+
+        LogLevel loglevel;
+        switch (level) {
+            case LOGSEVERITY_DISABLE:
+                return false;
+            case LOGSEVERITY_ERROR:
+                loglevel = LogLevel::Warn;
```
Comment (ResearchDaniel) [inline modules/webbrowser/src/webbrowserclient.cpp]: Error?

**Review by ResearchDaniel — CHANGES_REQUESTED** [review]
Nice, I’ve been looking for this functionality :)
Error should should have a different log audience?

File: modules/webbrowser/src/webbrowserclient.cpp [hunk 2]
```diff
@@ -227,4 +226,41 @@ void WebBrowserClient::OnLoadError(CefRefPtr<CefBrowser> browser, CefRefPtr<CefF
...
+            case cef_log_severity_t::LOGSEVERITY_DEBUG:
+            case cef_log_severity_t::LOGSEVERITY_INFO:
+            case cef_log_severity_t::LOGSEVERITY_DEFAULT:
+            default:
+                loglevel = LogLevel::Info;
+                break;
+        }
+
+        lc->log("WebBrowserClient", loglevel, LogAudience::Developer, file.c_str(), "", line,
+                message.ToString());
```
Comment (ResearchDaniel) [inline modules/webbrowser/src/webbrowserclient.cpp:261]: return true afterwards since we handle the logging?

Comment (r-englund) [reply modules/webbrowser/src/webbrowserclient.cpp:261]: fixed

File: modules/webbrowser/src/webbrowserclient.cpp (hunk 1 above)
Comment (r-englund) [reply modules/webbrowser/src/webbrowserclient.cpp]: fixed

**Review by ResearchDaniel — APPROVED** [review]

**Pull request merged**